R/sqlCreateTableAs.R

```diff
@@ -31,7 +31,11 @@ setGeneric("sqlCreateTableAs",
 #' @rdname PrestoConnection-class
 #' @usage NULL
 .sqlCreateTableAs <- function(con, name, sql, with = NULL, ...) {
-  name <- DBI::dbQuoteIdentifier(con, name)
+  if (inherits(name, "dbplyr_table_path")) { # dbplyr >= 2.5.0
+    name <- dbplyr::table_path_name(name, con)
+  } else {
+    name <- DBI::dbQuoteIdentifier(con, name)
```

Comment:
I think this is still breaking for dbplyr < 2.5.0 and DBI >= 1.2.0

Reply:
Can you be more specific about the breaking? I ran all the unit tests with dbplyr 2.4.0 and DBI 1.2.3 and all tests passed.